ui/components/app/modals/contract-details-modal/index.scss

```diff
@@ -0,0 +1,31 @@
+.contract-details-modal {
+  width: 360px !important;
+
+  &__content {
+    border-bottom: 1px solid var(--color-border-muted);
+
+    &__contract {
+      &__identicon {
+        margin: 16px 16px 38px 16px;
+      }
+
+      &__buttons {
+        flex-grow: 1;
+
+        &__copy.btn-link {
+          padding: 0;
+        }
+
+        &__block-explorer.btn-link {
+          padding: 0;
+        }
+      }
+    }
+  }
+
+  &__footer {
+    button + button {
+      margin-left: 1rem;
```

Comment:
`margin-inline-start` would be better for RTL

Reply:
Hi @darkwing Done [here](https://github.com/MetaMask/metamask-extension/pull/15549/commits/c7c6bd991d609b3544b8a5df4a12692ccca100b1#diff-7258ea0ef1005b16933bd9083a0d3013abc97dbc7ec498286b54a5e7633c7d75).